spec/unit/provider/user/linux_spec.rb

```diff
@@ -70,4 +72,54 @@
       expect( provider.useradd_options ).to eql(["-m"])
     end
   end
+
+  describe "expire_date behavior" do
```

Comment:
Sorry, these options are in `universal_options`, not `useradd_options`, my bad. This diff makes your tests pass for me:

```ruby
[phil@rider (Stromweld-master) RVM(2.7.1) chef]$ git diff
diff --git a/spec/unit/provider/user/linux_spec.rb b/spec/unit/provider/user/linux_spec.rb
index 8c2a7c52cd..61d7c43c6a 100644
--- a/spec/unit/provider/user/linux_spec.rb
+++ b/spec/unit/provider/user/linux_spec.rb
@@ -84,17 +84,17 @@ describe Chef::Provider::User::Linux do
     end
 
     it "by default expire_date is nil and we use ''" do
-      expect( provider.useradd_options ).to eql([""])
+      expect( provider.universal_options ).to eq([])
     end
 
     it "setting expire_date to nil includes ''" do
       @new_resource.expire_date nil
-      expect( provider.useradd_options ).to eql([""])
+      expect( provider.universal_options ).to eq([])
     end
 
     it "setting expire_date to 1982-04-16 includes -e" do
       @new_resource.expire_date "1982-04-16"
-      expect( provider.useradd_options ).to eql(["-e"])
+      expect( provider.universal_options ).to eq(["-e", "1982-04-16"])
     end
   end
 
@@ -109,17 +109,17 @@ describe Chef::Provider::User::Linux do
     end
 
     it "by default inactive is nil and we use ''" do
-      expect( provider.useradd_options ).to eql([""])
+      expect( provider.universal_options ).to eq([])
     end
 
     it "setting inactive to nil includes ''" do
       @new_resource.inactive nil
-      expect( provider.useradd_options ).to eql([""])
+      expect( provider.universal_options ).to eq([])
     end
 
     it "setting inactive to 90 includes -f" do
       @new_resource.inactive 90
-      expect( provider.useradd_options ).to eql(["-f"])
+      expect( provider.universal_options ).to eql(["-f", 90])
     end
   end
 end
```

Reply:
Thanks, that worked and all unit tests are passing now.